roles/engineblock/tasks/test.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Create install directory
+  file:
+    path: "{{ engineblock_selenium_install_dir }}/selenium"
+    state: directory
+    recurse: yes
+
+- name: Install Selenium standalone
+  get_url:
+    url: "http://selenium-release.storage.googleapis.com/{{ engineblock_selenium_version | regex_replace('\\.[0-9]+$', '') }}/selenium-server-standalone-{{ engineblock_selenium_version }}.jar"
+    dest: "{{ engineblock_selenium_install_dir }}/selenium/selenium-server-standalone-{{ engineblock_selenium_version }}.jar"
+    mode: 0755
+
+- name: Install ChromeDriver
+  unarchive:
+    src: "https://chromedriver.storage.googleapis.com/{{ engineblock_chromedriver_version }}/chromedriver_linux64.zip"
+    dest: "{{ engineblock_selenium_install_dir }}/selenium"
+    mode: 0755
+    copy: no
+
+- name: Install Chrome
+  yum:
+    name: https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
+    state: present
+
```

Comment:
You could also opt to install chromedriver from the local YUM repositories: It will add chromium as a dependency

Reply:
I've considered using yum but chrome is also used for frontend testing and using the latest and greatest would prevent much incompatibility issues there.